MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -101,16 +104,21 @@ void PresentBudgetChart(IEnumerable<Category> categories)
                 LabelMode = LabelMode.RightOnly
             };
         }
-        void PresentTransactionChart(IEnumerable<Transaction> transactions)
+void PresentTransactionChart(IEnumerable<Transaction> transactions)
         {
             List<ChartEntry> entries = new List<ChartEntry>();
             try
             {
                 foreach (var iteam in transactions)
                 {
-                        entries.Add(new ChartEntry((float)iteam.Amount)
-                        {
-                        }
+                    net += (float)iteam.Amount;
+                    if ((float)iteam.Amount > 0)
+                        income += (decimal)iteam.Amount;
+                    else
+                        expenses += (decimal)iteam.Amount;
+                    entries.Add(new ChartEntry(net)
+                    {
+                    }
```

Comment:
Why these brackets are needed?

Reply:
It's for entries to microcharts. For now they're empty because line chart doesn't need labels, colors, etc.